packages/twenty-front/src/modules/settings/data-model/components/SettingsObjectFieldPipelineStepsFormOptionRow.tsx

```diff
@@ -0,0 +1,172 @@
+import { useMemo } from 'react';
+import { useTheme } from '@emotion/react';
+import styled from '@emotion/styled';
+import { v4 } from 'uuid';
+
+import { ColorSample } from '@/ui/display/color/components/ColorSample';
+import {
+  IconCheck,
+  IconDotsVertical,
+  IconGripVertical,
+  IconTrash,
+  IconX,
+} from '@/ui/display/icon';
+import { LightIconButton } from '@/ui/input/button/components/LightIconButton';
+import { TextInput } from '@/ui/input/components/TextInput';
+import { Dropdown } from '@/ui/layout/dropdown/components/Dropdown';
+import { DropdownMenu } from '@/ui/layout/dropdown/components/DropdownMenu';
+import { DropdownMenuItemsContainer } from '@/ui/layout/dropdown/components/DropdownMenuItemsContainer';
+import { useDropdown } from '@/ui/layout/dropdown/hooks/useDropdown';
+import { MenuItem } from '@/ui/navigation/menu-item/components/MenuItem';
+import { MenuItemSelectColor } from '@/ui/navigation/menu-item/components/MenuItemSelectColor';
+import { mainColorNames } from '@/ui/theme/constants/colors';
+
+import { SettingsObjectFieldPipelineStepsFormOption } from '../types/SettingsObjectFieldPipelineStepsFormOption';
+
+type SettingsObjectFieldPipelineStepsFormOptionRowProps = {
+  className?: string;
+  isDefault?: boolean;
+  onChange: (value: SettingsObjectFieldPipelineStepsFormOption) => void;
+  onRemove?: () => void;
+  option: SettingsObjectFieldPipelineStepsFormOption;
+  index: number;
+};
+
+const StyledRow = styled.div`
+  align-items: center;
+  display: flex;
+  height: ${({ theme }) => theme.spacing(6)};
+  padding: ${({ theme }) => theme.spacing(1.5)} 0;
+`;
+
+const StyledColorSample = styled(ColorSample)`
+  cursor: pointer;
+  margin-left: 9px;
+  margin-right: 14px;
```

Comment:
This margin-left 9px and margin-right 14px looks suspicious! Double check the figma to see what we really want and use theme.spacing()